sdk-commons/src/main/java/org/openmrs/maven/plugins/model/Server.java

```diff
@@ -67,6 +70,8 @@ public class Server extends BaseSdkProperties {
 
     public static final String PROPERTY_SERVER_PORT = "server.port";
 
+    private static final Logger log = LoggerFactory.getLogger(Server.class);
```

Comment:
What are we using this for?

Reply:
First I decided to log the error message rather than throwing the exception. I forgot to remove it after changing the code. I removed this in the new push.